lib/utils.py

```diff
@@ -351,3 +355,8 @@ def str_to_bytes(s):
     if PY2:
         return s
     return s.encode('utf-8')
+
+
+def binary_regex(s, flags=0):
```

Comment:
I don't think this function is really useful for two reasons:
1) It adds 3 unnecessary function calls (`binary_regex`, `str_to_bytes` and `encode`) and doesn't reduce the amount of code.
2) It's so simple, just write `re.compile (b" ")`.

Reply:
As I said above it is not so easy due to the "rb" prefix. Our patterns are raw strings. Either we should avoid raw strings and use b-prefix, or use this method.